tracetools_trace/tracetools_trace/tools/args.py

```diff
@@ -84,8 +84,12 @@ def _add_arguments_configure(parser: argparse.ArgumentParser) -> None:
         '--live', dest='live_timer_interval', type=int, nargs='?',
         # Default value for 'lttng create-session --live':
         # https://lttng.org/man/1/lttng-create/v2.13/#doc-opt--live
-        default=1000000,
-        help='TODO (default: %(default)s)')
+        const=100000,
+        help='Set the live timer interval (default: %(default)s)')
```

Comment:
This does set the live timer interval, but the main use of the `--live` flag is to create a live tracing session, so this should be something like:

```suggestion
        help='Create a live tracing session. Optionally set the live timer interval (default: %(default)s)')
```

(double check the line length and split it over two lines if necessary)

Reply:
Done.